downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/DownloadConfirmationFragment.kt

```diff
@@ -82,6 +84,7 @@ class DownloadConfirmationFragment : BottomSheetDialogFragment() {
     }
 
     private fun setupViews(binding: DownloadConfirmationBinding) {
+        (dialog as BottomSheetDialog).behavior.state = BottomSheetBehavior.STATE_EXPANDED
```

Comment:
With this change, corners won't be rounded anymore even in portrait mode. Therefore, I would apply this state only if the orientation is landscape if you agree. I tried to find a quick solution where the corners would remain rounded even in the expanded mode by default, but I couldn't find one so I think this would be enough to fix this issue just now.
```suggestion
        if (resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE) {
            (dialog as BottomSheetDialog).behavior.state = BottomSheetBehavior.STATE_EXPANDED
        }
```